.asf.yaml

```diff
@@ -35,6 +35,4 @@ github:
         strict: true
         contexts:
           - CheckStatus
-      required_pull_request_reviews:
-        dismiss_stale_reviews: true
-        required_approving_review_count: 1
```

Comment:
I'm removing this since trivial changes like this PR can be merged quickly as long as the checks passed

Reply:
How about non-trivial changes ?

Reply:
> How about non-trivial changes ?

Committers can request reviews from other committers, as we have limited committers in this repo, I'd rather work on trust that we know the changes are non-trivial and need review

Reply:
Can those people who are not members (committer/PMC) merge directly when the checks passed ? If they can't I think this change of `.asf.yml` is ok.

Reply:
> Can those people who are not members (committer/PMC) merge directly when the checks passed ? If they can't I think this change of .asf.yml is ok.

Of course they can't, merging needs write access to the repo

Reply:
Ok, I have no worries.